src/main/common/components/ccj/paid-amount-summary.njk

```diff
@@ -8,9 +8,8 @@
   <div class="column-two-thirds">
 
     <div class="form-group">
-      <p>
-        {{ t('Judgment will be entered against the defendant for this amount.') }}
```

Comment:
Why is this being removed? It is in the prototype: 
https://hmcts-cmc-prototype.herokuapp.com/prototypes/features-to-be-built/dashboard/applications-and-judgments/default-judgment/claim-amount?radio-indent-group2=Yes&num2=500

Reply:
Marc asked for it to be removed